<?xml version="1.0"?>

<launch>
  

  <node pkg="pointcloud_filter" type="pointcloud_cluster_with_mask" name="mask_filter" output="screen">
    <remap from="cloud_input" to="velo_cloud" />
    <param name="base_frame" value="iMiev/icp" />
	<param name="map_frame" value="iMiev/map" />
	<!--<param name="map_location" value="$(find pointcloud_registration)/map/loam_utown_mask.pcd" />-->
	<param name="map_location" value="$(find pointcloud_filter)/mask/utown_road_mask.pcd" />
	<!--<param name="density" value="0.5" />-->
    
  </node>
  

</launch>
